docs/src/pages/demos/steppers/steppers.md

```diff
@@ -64,6 +62,12 @@ Labels can be placed below the step icon by setting the `alternativeLabel` prope
 
 {{"demo": "pages/demos/steppers/VerticalLinearStepper.js"}}
 
+## Customized Stepper
+
+The example also uses a customized `StepConnector` element that changes border color based on the `active` and `completed` state.
```

Comment:
The text is a bit off now that we moved to its own section. Maybe

```
This component uses a customized `StepConnector` element that changes border color based on the `active` and `completed` state.
```

Reply:
Good idea!